package.json

```
@@ -25,7 +25,7 @@
     "src/*"
   ],
   "dependencies": {
-    "@holo-host/chaperone": "^0.2.2",
+    "@holo-host/chaperone": "Holo-Host/chaperone#B-03916-first-time-login",
```

In my experience branch dependencies can't be updated without deleting package-lock.json or manually editing it. Therefore I suggest a revision dependency instead

```suggestion
    "@holo-host/chaperone": "Holo-Host/chaperone#b5a11a3dd0e0c0e15430e48ac507df33ada0ecc8",
```

---

Very true; great suggestion.